components/dashboard/src/components/Modal.tsx

```diff
@@ -50,9 +50,9 @@ export default function Modal(props: {
     return (
         <div className="fixed top-0 left-0 bg-black bg-opacity-70 z-50 w-screen h-screen" onClick={props.onClose}>
             <div className="w-screen h-screen align-middle" style={{display: 'table-cell'}}>
-                <div className={"relative bg-white border rounded-xl p-6 max-w-lg mx-auto text-left text-gray-600 " + (props.className || '')} onClick={e => e.stopPropagation()}>
+                <div className={"relative bg-white dark:bg-gray-900 border border-gray-200 dark:border-gray-700 rounded-xl p-6 max-w-lg mx-auto text-left text-gray-600 " + (props.className || '')} onClick={e => e.stopPropagation()}>
                     {props.closeable !== false && (
-                        <div className="absolute right-7 top-6 cursor-pointer hover:bg-gray-200 rounded-md p-2" onClick={props.onClose}>
+                        <div className="absolute right-7 top-6 cursor-pointer hover:bg-gray-200 dark:hover:bg-gray-700 rounded-md p-2" onClick={props.onClose}>
                             <svg version="1.1" width="14px" height="14px"
```

Comment:
**issue**: This should be using a light color (`gray-100`) like the title next to it.